src/custom-widget.ts

```diff
@@ -1,28 +1,16 @@
 import * as shortid from "shortid";
-import {
-  ITagoIO,
-  ICallbackStart,
-  IError,
-  IData,
-  IVariable,
-  ICallbackRealtime,
-  ICallbackError,
-  ITagoVariables,
-  IEvent,
-  IMessage,
-} from "./interfaces";
-import { enableAutofill } from "./utils";
+import { autoFillRecords } from "./utils";
```

Comment:
Renamed this function since it seemed confusing that `enable` actually was "auto-filling" instead of "enabling" something.